plugins/check_port/init.js

```diff
@@ -46,41 +45,20 @@ plugin.createPortMenu = function(e)
 	return(false);
 }
 
-plugin.toggleIpAddress = function()
-{
-	const ipAddr = $("#port-ip-text");
-	const ipPane = $("#port-pane");
-	if (ipAddr.hasClass("censored")) {
-		ipAddr.text("*.*.*.*:*");
-		ipPane.prop(
-			"title", `${ipAddr.text()} - ${theUILang.portStatus[ipPane.data("status")]} (${theUILang.clickReveal})`
-		);
-	} else {
-		ipAddr.text(ipPane.data("ip") + ":" + ipPane.data("port"));
-		ipPane.prop(
-			"title", `${ipAddr.text()} - ${theUILang.portStatus[ipPane.data("status")]} (${theUILang.clickHide})`
-		);
-	}
-}
-
 plugin.onLangLoaded = function()
 {
 	plugin.addPaneToStatusbar(
 		"port-pane",
 		$("<div>").append(
 			$("<div>").addClass("icon"),
-			$("<span>").attr("id","port-ip-text"),
+			$("<span>").attr("id","port-ip-text").addClass("d-none d-lg-block"),
```

Comment:
IP address is hidden on small screens and begin to show up on large screens (>=992px wide).